Makefile

```diff
@@ -57,8 +59,8 @@ check_config_status = \
 	fi;
 
 deps-check:
-	$(call check_pkg_config,Magick++,libmagick++-dev)
-	$(call check_pkg_config,ImageMagick++,libgraphicsmagick++-dev)
```

Comment:
Why the `ImageMagick++` -> `GraphicsMagick++` change?

Reply:
Because that check was wrong to begin with. `Magick++` and `ImageMagick++` both belong to libmagick++, whereas `GraphicsMagick++` is the pkg-config name of `libgraphicsmagick++1`.